openreview/venue/process/invite_assignment_pre_process.js

```diff
@@ -3,14 +3,15 @@ async function process(client, edge, invitation) {
 
   const { groups } = await client.getGroups({ id: edge.domain })
   const domain = groups[0]
-  const quota = domain.content.reviewer_assignment_quota?.value
   const reviewersId = invitation.content.match_group?.value
   const assignmentInvitationId = invitation.content.assignment_invitation_id?.value
   const conflictInvitationId = invitation.content.conflict_invitation_id?.value
   const assignmentLabel = invitation.content.assignment_label?.value
   const inviteLabel = invitation.content.invite_label?.value
   const conflictPolicy = domain.content.reviewers_conflict_policy?.value
   const conflictNYears = domain.content.reviewers_conflict_n_years?.value
+  const reviewersName = reviewersId.split('/').pop().toLowerCase()
```

Comment:
there is a property called `reviewers_name` that you can use, no need to split the group id.

Reply:
I can't call that directly right because this is used for other roles as well?